library/src/main/java/com/bumptech/glide/load/data/mediastore/MediaStoreUtil.java

```diff
@@ -20,6 +20,10 @@ public static boolean isMediaStoreUri(Uri uri) {
         && MediaStore.AUTHORITY.equals(uri.getAuthority());
   }
 
+  public static boolean isAndroidPickerUri(Uri uri) {
+    return isMediaStoreUri(uri) && uri.getPathSegments().contains("picker");
```

Comment:
Any chance this constant comes from anywhere that could be referenced? Or a page we could link to?